fio/lib/src/utils.ts

```diff
@@ -0,0 +1,281 @@
+    abi_hash:
+      "19a099047a11ee6c2a1d86b44b9100eba44ef85b60e33edfbdfcc58ff6ab7c56",
+    account_name: "fio.address",
+    code_hash:
+      "72c258fbcb0b328960bdbc4fc521d76cc53e5517bb5f5f2bf991d08030ddae6b",
+  };
+
+  const abiMap = new Map<string, FioTokenAbi>();
+  abiMap.set(account, fioTokenAbi);
+
+  const abiProvider: AbiProvider = {
+    getRawAbi: async () => {
```

Comment:
This interface doesn't look right. See eos/fio/frontend/node_modules/@fioprotocol/fiojs/dist/chain-api-interfaces.d.ts
<img width="926" alt="image" src="https://github.com/PassbirdCo/MetaKeepCodeSamples/assets/100057389/31e37a3a-3482-4858-a72d-788e0ebbf053">

Reply:
Done